code/modules/modular_computers/hardware/portable_hard_drive.dm

```diff
@@ -33,6 +33,21 @@
 	max_capacity = 256
 	origin_tech = list(TECH_DATA = 4)
 
+/obj/item/weapon/computer_hardware/hard_drive/portable/manual
```

Comment:
switch this path to /obj/item/weapon/computer_hardware/hard_drive/portable/autorun/manual

Have the auto transfer and run behaviour on the autorun type